css-brochure-form/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="form-box">
+      <h2>LEARN MORE</h2>
+      <form>
+        <div class="form-box">
```

Comment:
Again, the `form-box` class sets a width of `58%` and `margin: auto`, I don't know why those would be necessary on these divs, the width should only be necessary for the most external `div` holding all the content.

Reply:
I moved width and margin:auto from the form-box class ruleset and put them into the body ruleset since it holds all the content. Should I make a new div that has a class name "container" or something along those lines and incorporate the width and margin in that ruleset instead of the body? or is it fine in the body ruleset

Reply:
I guess what I want to know is should I use the margin: auto to center my content on the webpage like I have been? and should I use the width to finetune the input boxes' width? Or should I go with a different set of property/value pairs?